tests/scripts/generate_test_keys.py

```diff
@@ -89,12 +90,16 @@ def get_look_up_table_entry(key_type: str, group_id_or_keybits: str,
     yield "      {0}, sizeof({0}),\n".format(priv_array_name)
     yield "      {0}, sizeof({0}) }},".format(pub_array_name)
 
+#pylint: disable=too-many-locals
 def main() -> None:
```

Comment:
Please bypass pylint checks only at the narrowest practical level. Putting the directive at the toplevel applies to all subsequent functions as well.
```suggestion
def main() -> None:
    #pylint: disable=too-many-locals
```
Ideally we'd split this function: I think pylint's warning is justified here. But it was already just barely under the threshold before, so that is preexisting and out of scope.